src/js/component.js

```diff
@@ -999,6 +999,20 @@ class Component {
   }
 
   /**
+   * Set the focus to this component
+   */
+  focus() {
+    this.el_.focus();
+  }
```

Comment:
I think we already have these in master? Does this PR need a rebase for some reason? https://github.com/videojs/video.js/blob/master/src/js/component.js#L1004